src/components/MicrofactsTooltip/MicrofactsTooltip.tsx

```diff
@@ -12,113 +12,111 @@ const useStyles = makeStyles((theme: Theme) =>
+const Factbox = (props: FactboxProps) => {
   const classes = useStyles();
-  const { entity, updateEntity } = props;
+  const { entity, description, displayLink, link } = props;
+
+  return (
+    <Card className={classes.factBoxCard}>
+      <Grid container direction="column" spacing={0.5}>
+        <Grid item>
+          <Typography variant="h6">{entity}</Typography>
+        </Grid>
+        <Grid item>
+          <Typography variant="body1" fontSize={'0.9em'}>
+            {description.length > 220
+              ? description.slice(0, 220) + ' ...'
+              : description}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Typography fontFamily="DM Mono" fontSize={'0.8em'}>
+            Read more on:{' '}
+            <a href={link} target="_blank" rel="noreferrer">
+              {displayLink}
+            </a>
+          </Typography>
+        </Grid>
+      </Grid>
+    </Card>
+  );
+};
 
-  const onCloseFeedback = () => {
-    if (
-      entity != undefined &&
-      entity.feedbackIsHelpful != undefined &&
-      entity.hasGivenFeedback != undefined &&
-      entity.hasGivenFeedback
-    )
-      updateEntity(entity, { ...entity, showFeedback: false });
-  };
+export interface MicrofactsTooltipProps {
+  microfact: Entity;
+  keepTooltipInsideId?: string;
+}
 
-  const showMicrofact =
-    // entity has to be defined
-    entity != undefined &&
-    // entity has to be selected to be shown
-    entity.checked &&
-    // entity is a key fact or all facts should be shown
-    entity.keyFact;
+export const MicrofactsTooltip = (props: MicrofactsTooltipProps) => {
+  const classes = useStyles();
+  const { microfact, keepTooltipInsideId } = props;
 
-  const getMaxLengthDescription = () => {
-    const sentences = entity?.description.split('. ') ?? [];
-    let result = '';
-    for (let i = 0; i < 6 && i < sentences.length; i++) {
-      result = result.concat(sentences[i], '. ');
-    }
-    return result.slice(0, result.length - 2) + '..';
-  };
+  const capitalizeSentence = (sentence: string) =>
+    sentence.charAt(0).toUpperCase() + sentence.slice(1);
 
   return (
     <>
-      {showMicrofact && (
+      {microfact.checked ? (
         <Popup
           trigger={() => (
-            <Box
-              style={{ backgroundColor: '#FFF9DA', cursor: 'pointer' }}
-              component="span"
-            >
-              {entity.entity}
+            <Box component="span" className={classes.triggerBox}>
+              {microfact.text}
             </Box>
           )}
           position={[
             'bottom center',
-            'bottom right',
             'bottom left',
-            'left top',
+            'bottom right',
             'right top',
+            'top left',
+            'top right',
           ]}
-          onClose={onCloseFeedback}
-          on={['click', 'focus']}
+          closeOnDocumentClick
+          keepTooltipInside={keepTooltipInsideId}
```

Comment:
I think it looks better without the arrow, you can set `arrow={false}` here.

Reply:
Fixed